documentation/docs/60-appendix/05-integrations.md

```diff
@@ -31,4 +31,4 @@ You will need to install `svelte-preprocess` with `npm install --save-dev svelte
 
 ## Integration FAQs
 
-The SvelteKit FAQ has a [section on integrations](../faq#integrations), which may be helpful if you still have questions.
+The SvelteKit FAQ has a [section on integrations](./faq#how-do-i-use-x-with-sveltekit), which may be helpful if you still have questions.
```

Comment:
```suggestion
The SvelteKit FAQ has a [how to do X with SvelteKit](./faq#how-do-i-use-x-with-sveltekit), which may be helpful if you still have questions.
```

This link might make a bit more sense for this page

Reply:
Ready!